File: seeds/javascript/test.solution.mjs
import assert from "node:assert";
import 
{
	entrypoint;
}
 from './solution.mjs'

/*
REMEMBER:
- Your tests should be comprehensive. They should test 
  - edge cases 
  - large inputs 
  - as many scenarios as possible.
- Minimum 5 tests. Maximum 10 tests.
- More tests, higher-quality tests === higher payouts.
*/

assert.strictEqual({ entrypoint }(/* args */) /* expected result */);

assert.strictEqual({ entrypoint }(/* args */) /* expected result */);

assert.strictEqual({ entrypoint }(/* edge case args */) /* expected result */);

assert.strictEqual({ entrypoint }(/* large input */) /* expected result */);

try {
	entrypoint;
	(/* invalid args */);
	assert.fail("Expected error was not thrown");
} catch (error) {
	assert.strictEqual(error.message, "Expected error message");
}

assert.strictEqual({ entrypoint }(/* args */) /* expected result */);

File: seeds/python/test_solution.py
import pytest
from solution import {entrypoint}

"""
REMEMBER:
- Your tests should be comprehensive. They should test 
  - edge cases 
  - large inputs 
  - as many scenarios as possible.
- Minimum 5 tests. Maximum 10 tests.
- More tests, higher-quality tests === higher payouts.
"""


def test_one():
    """
    # Example assertion:
    assert {entrypoint}(
        arg1=value1,
        arg2=value2
    ) == expected_result
    
    Rename this function to test_{description of the test}, e.g. test_basic_graph_traversal
    """
    assert {entrypoint}(# args) == # expected result

def test_two():
    """
    # Example assertion:
    assert {entrypoint}(
        arg1=value1,
        arg2=value2
    ) == expected_result
    
    Rename this function to test_{description of the test}, e.g. test_graph_traversal_with_multiple_paths
    """
    assert {entrypoint}(# args) == # expected result

def test_three():
    """
    # Example assertion:
    assert {entrypoint}(
        arg1=value1,
        arg2=value2
    ) == expected_result
    
    Rename this function to test_{description of the test}, e.g. test_invalid_graph_input
    """
    assert {entrypoint}(# edge case args) == # expected result

def test_four():
    """
    # Example assertion:
    assert {entrypoint}(
        arg1=value1,
        arg2=value2
    ) == expected_result
    
    Rename this function to test_{description of the test}, e.g. test_massive_graph
    """
    assert {entrypoint}(# large input) == # expected result

def test_five():
    """
    # Example assertion:
    with pytest.raises(ValueError):
        {entrypoint}(invalid_args)
    
    Rename this function to test_{description of the test}, e.g. test_proper_memoization
    """
    try:
        {entrypoint}(# invalid args)
        assert False, "Expected error was not raised"
    except ValueError as e:
        assert str(e) == "Expected error message"

    assert {entrypoint}(# args) == # expected result 